--- vokoscreen.pro.orig	2016-08-16 21:20:23.012199582 +0200
+++ vokoscreen.pro	2016-08-16 21:23:07.219893639 +0200
@@ -26,7 +26,7 @@
 # Um der Fehlermeldung entgegenzuwirken das keine *.qm Dateien vorhanden sind wird lrelease als Systemaufruf vorher aufgerufen.
 # Das Script/Macro siehe weiter unten "# language packages" muß weiter bestehen bleiben damit "make clean" die *.qm Dateien löscht.
 
-system(lrelease language/vokoscreen_*.ts)
+system(lrelease-qt4 language/vokoscreen_*.ts)
           
 RESOURCES += screencast.qrc
                         
@@ -36,10 +36,10 @@
 !isEmpty(TRANSLATIONS) {
   isEmpty(QMAKE_LRELEASE) {
     win32: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]\lrelease.exe
-      else: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
+      else: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease-qt4
   }
   isEmpty(TS_DIR):TS_DIR = language
-  TSQM.name = lrelease ${QMAKE_FILE_IN}
+  TSQM.name = lrelease-qt4 ${QMAKE_FILE_IN}
   TSQM.input = TRANSLATIONS
   TSQM.output = $$TS_DIR/${QMAKE_FILE_BASE}.qm
   TSQM.commands = $$QMAKE_LRELEASE ${QMAKE_FILE_IN}
